shell/browser/api/atom_api_screen.cc

```diff
@@ -99,17 +99,38 @@ static gfx::Rect DIPToScreenRect(electron::NativeWindow* window,
 
 #endif
 
+void DelayEmit(Screen* screen,
+               const base::StringPiece& name,
+               const display::Display& display) {
+  screen->Emit(name, display);
+}
+
+void DelayEmitWithMetrics(Screen* screen,
+                          const base::StringPiece& name,
+                          const display::Display& display,
+                          std::vector<std::string> metrics) {
```

Comment:
`const std::vector<std::string>&`? it should get copied by `PostTask`, no need to copy it again when the task queue calls the function.